binaries/upload.py

```diff
@@ -45,12 +45,6 @@ def upload_pypi_packages(args, WHL_PATHS):
         required=False,
         help="Specify whether to upload pypi packages",
     )
-    parser.add_argument(
-        "--test-pypi",
-        action="store_true",
-        required=False,
-        help="Specify whether to upload to test PyPI",
-    )
```

Comment:
It seems the wrong section is deleted.
Maybe we want to delete `--upload-pypi-packages` (not used) and keep `--test-pypi` (used)?